mono/metadata/reflection.c

```diff
@@ -11784,15 +11801,22 @@ mono_reflection_create_runtime_class (MonoReflectionTypeBuilder *tb)
 			MonoReflectionMethodBuilder *mb = (MonoReflectionMethodBuilder *)mono_array_get (tb->methods, gpointer, i);
 			if (mb) {
 				RESOLVE_TYPE (mb->rtype);
-				check_array_for_usertypes (mb->return_modreq);
-				check_array_for_usertypes (mb->return_modopt);
-				check_array_for_usertypes (mb->parameters);
+				check_array_for_usertypes (mb->return_modreq, &error);
```

Comment:
Those long chains of call+throw or call+check are fugly, we'll have to handle them at some point.